jvm/src/main/scala/eval/tile/LazyRaster.scala

```diff
@@ -142,6 +141,28 @@ object LazyRaster {
     def getDouble(col: Int, row: Int) = f(fst.getDouble(col, row), scalar)
   }
 
+  case class Convert(
+    children: List[LazyRaster],
+    cellType: CellType
+  ) extends UnaryBranch {
+    lazy val intTile = fst.evaluate.convert(cellType)
```

Comment:
Should be removed because it makes no sense for this node to be here and it would make a lazy MAML not lazy at all.